### DISCUSSION TITLE
Intro GIS Propraming

### DISCUSSION THREAD

**Discussion by HongchhayChheng** [giswqs/geog-312#13] [Q&A] [answered]
![Screenshot 2024-08-21 094728](https://github.com/user-attachments/assets/8514289a-2844-4e83-a8e0-3fb46a6e07e6)
Dear Prof, I have some problems with Miniconda may you take your time to check my problem teacher. 
Thank you in advance

**Comment by giswqs** [comment, marked as answer]
Use Anaconda Prompt from the Start Menu.

**Reply by HongchhayChheng** [reply]
I get it, Thank you so much for responding Prof.
